backend/src/api/mining.ts

```diff
@@ -62,12 +62,30 @@ class Mining {
     }
 
     const blockCount: number = await BlocksRepository.$blockCount(pool.id);
-    const emptyBlocksCount = await BlocksRepository.$countEmptyBlocks(pool.id);
+    const totalBlock: number = await BlocksRepository.$blockCount(null, null);
+
+    const blockCount24h: number = await BlocksRepository.$blockCount(pool.id, '24h');
+    const totalBlock24h: number = await BlocksRepository.$blockCount(null, '24h');
+
+    const blockCount1w: number = await BlocksRepository.$blockCount(pool.id, '1w');
+    const totalBlock1w: number = await BlocksRepository.$blockCount(null, '1w');
+
+    const currentEstimatedkHashrate = await bitcoinClient.getNetworkHashPs(totalBlock24h);
```

Comment:
I am concerned about adding direct calls to Bitcoin RPC from a public API call as Bitcoin RPC only can handle as much. If it will be handled with some Nginx cache I guess it will be fine?

Reply:
Yes, `/api/v1/mining/pool/{slug}` is cache warmed. HTTP header `Expires` is set to 1 minute in the node backend right now.